pandas/tests/extension/test_interval.py

```diff
@@ -83,7 +83,9 @@ class TestCasting(BaseInterval, base.BaseCastingTests):
 
 
 class TestConstructors(BaseInterval, base.BaseConstructorsTests):
-    pass
+    @pytest.mark.xfail(reason="bad is-na for empty data")
```

Comment:
why is this xfailed?

Reply:
- object is not supported for IntervalArray
- na_value of IntervalArray is float, so `AttributeError: 'float' object has no attribute 'dtype'` in construct_1d_arraylike_from_scalar().

Reply:
I think this would ideally fixed here.

Reply:
if nan_dtype is dtype (IntervalDtype), can create df.
```
if is_interval_dtype(dtype):
    nan_dtype = dtype
```